README.md

```diff
@@ -17,19 +17,17 @@ Implement the `Session` entity:
 
 import { ISession } from "connect-typeorm";
 import { Column, Entity, Index, PrimaryColumn } from "typeorm";
-import { Bigint } from "typeorm-static";
-
 @Entity()
 export class Session implements ISession {
   @Index()
-  @Column("bigint", { transformer: Bigint })
+  @Column("bigint")
   public expiredAt = Date.now();
 
   @PrimaryColumn("varchar", { length: 255 })
-  public id = "";
+  public id: string;
```

Comment:
Explicit field initializers are needed for apps to pass TypeScript strict property checks, and will help you avoid runtime TypeError. Please keep them in the example.

Reply:
Ah! strict mode!